package com.luciano.cadastropessoa.cadastrarpessoa.exception

class AuthorNotFoundException(idCountry: Long) : RuntimeException("N√ÉO EXISTE ESSE ID TESTE ERRO: $idCountry")
